src/module/rules/rule-element/item-alteration/schemas.ts

```diff
@@ -301,6 +301,37 @@ const ITEM_ALTERATION_VALIDATORS = {
         }),
         value: new StrictNumberField({ required: true, nullable: false, integer: true, initial: undefined } as const),
     }),
+    group: new ItemAlterationValidator(
+        {
+            itemType: new fields.StringField({ required: true, choices: ["armor", "weapon"] }),
+            mode: new fields.StringField({ required: true, choices: ["override"] }),
+            value: new fields.StringField({
+                required: true,
+                nullable: false,
+                initial: undefined,
+            } as const),
+        },
+        {
+            validateForItem: (item, alteration): validation.DataModelValidationFailure | void => {
+                const value = alteration.value;
+                if (item.type === "armor") {
+                    const validTraits = CONFIG.PF2E.armorGroups;
+                    if (typeof value !== "string" || !(value in validTraits)) {
```

Comment:
You should be able to replace this with `objectHasKey(CONFIG.PF2E.armorGroups, value)`, which was added to make these checks simpler.